app/src/main/java/com/example/paetz/yacguide/utils/WidgetUtils.java

```diff
@@ -78,4 +78,16 @@ private static TextView _createTextView(Context context, String text, int textSi
 
         return textView;
     }
+
+    public static String getAscendSymbol(int styleId) {
+        String symbol = " ";
+        if (styleId < 9) {
+            symbol = "✓";
+        } else if (styleId == 9) {
+            symbol = "✗";
+        } else if (styleId < 14){
```

Comment:
Ascend styles "an den einstieg gepinkelt", "zugesehen" and "von gehört" should have an eye-symbol (1F441).

Reply:
> Ja ich denke, das ist sinnvoll. Folgendes fänd ich noch gut:
> 
>     1. Ich fänd es besser, wenn das Tourenbuch nur gekletterte Wege beinhaltet (Häkchen).
> 
>     2. Projekte (="will ich klettern") und Säcke sollten jeweils in separaten Listen sein, die man vom Hauptmenü aus erreicht. Die Symbole kannst du in diesen Listen dann weglassen.
> 
> 
> Das sollte man aber vllt als separaten PR machen. Willst du das auch noch machen oder soll ich?

Ich gucke es mir mal an und versuche mich mal dran.